packages/common/src/Utility.ts

```diff
@@ -587,3 +587,36 @@ export function removeHTMLFromString(str: string, div?: HTMLDivElement) {
     div.innerHTML = str;
     return div.textContent || div.innerText || "";
 }
+
+export function radiansToDegrees(radians: number): number {
+    return radians * 180 / Math.PI;
+}
+
+export function degreesToRadians(degrees: number): number {
+    return degrees * Math.PI / 180;
+}
+
+export function getPartialCircleDimensions(start: number, end: number, radius: number): {width: number, height: number} {
+    const firstAngle = degreesToRadians(start);
+    const lastAngle = degreesToRadians(end);
+    const x1 = radius * Math.cos(firstAngle);
+    const x2 = radius * Math.cos(lastAngle);
+    const y1 = radius * Math.sin(firstAngle);
+    const y2 = radius * Math.sin(lastAngle);
+    const point_arr = [];
+    if (firstAngle <= Math.PI * 0.5 && lastAngle >= Math.PI * 0.5) point_arr.push([      0,  radius]);
+    if (firstAngle <= Math.PI * 1.0 && lastAngle >= Math.PI * 1.0) point_arr.push([-radius,       0]);
+    if (firstAngle <= Math.PI * 1.5 && lastAngle >= Math.PI * 1.5) point_arr.push([      0, -radius]);
+    if (firstAngle <= Math.PI * 2.0 && lastAngle >= Math.PI * 2.0) point_arr.push([ radius,       0]);
+    point_arr.push([x1, y1]);
+    point_arr.push([x2, y2]);
+    point_arr.push([0, 0]);
+    const minX = Math.min.apply(undefined, point_arr.map(n => n[0]));
+    const minY = Math.min.apply(undefined, point_arr.map(n => n[1]));
+    const maxX = Math.max.apply(undefined, point_arr.map(n => n[0]));
+    const maxY = Math.max.apply(undefined, point_arr.map(n => n[1]));
+    return {
+        width: maxX - minX,
+        height: maxY - minY
+    };
+}
```

Comment:
I think these should be in @hpcc-js/util in a file called geometry?
Also in util, you will see everything has unit tests and a lot of stuff has some API documentation - which is probably a good idea for these types of utility?